packages/terra-i18n-plugin/docs/README.md

```diff
@@ -32,3 +32,10 @@ module.exports = {
 ```
 
 Then all of the messages for a given language across all dependencies will be aggregated into an overall translation file for that language and placed into an aggregated-translations directory.  In order for these resulting files to be able to be loaded by terra-i18n, that directory then needs to be added to the list of directories that the webpack resolver looks into.
+
+By default it will search translation folders in the root node_modules. The i18n-plugin will also search and aggregate translation files under a custom file path when the optional key `translationsDirectory` is specified:
+```js
+new I18nAggregatorPlugin({
+  translationsDirectory: 'customPath',
```

Comment:
What if we made this an array ( `translationsDirectories` ) that defaulted to `['node_modules']` but could be overridden to whatever you wanted `['customPath', 'node_modules']`. I think this could clean up some of the messy 'searchNodeModules' code below.

Reply:
That would be a non-passive change. But I am open to it since it aligns with some common webpack config format.

Reply:
We could keep it passive by assuming `translationsDirectories = ['node_modules']` if translationsDirectories are not provided. Am I missing something?

Reply:
@mjhenkes That's a good idea, then a consumer could provide multiple paths for 'custom' translation directories

Reply:
@mjhenkes the reason for adding `searchNodeModules` is that there are two different search strategies. For node_modules, it only searches the packages under node_modules, and then search the node_modules under each package. For customPath, it searches every possible path under it. So it has a larger scope. Is your idea of considering node_modules as a default customPath?

Reply:
@zhongyn utilizing this approach we'd have the ability to indicate more than one search pathern. For example say we specify 
```['node_modules', 'custom_translations', 'verified_translations']```
this would allow for searches for any `node_modules` folders, `custom_translations` folders and `verified_tranlsations `folders. This approach would allow for multi translation directories patterns to be search for and aggregated to a single file.

Reply:
updated config to support multi custom translation dirs